crates/holochain/src/conductor/conductor/tests.rs

```diff
@@ -716,6 +716,7 @@ async fn test_cells_disable_on_validation_panic() {
 }
 
 #[tokio::test(flavor = "multi_thread")]
+#[ignore = "Validation doesn't run from genesis so this test doesn't make sense anymore"]
```

Comment:
@maackle I disabled this test because I don't think it makes any sense anymore. Can you please confirm and maybe we can update or remove it?